modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux-using-rpm"]
+
+= Installing {pipelines-title} CLI (tkn) on Linux using an RPM
+
+For {op-system-base-full}, you can install the {pipelines-title} CLI (`tkn`) as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Prerequisites
+
+* Must have root or sudo privileges.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+----
+# subscription-manager list --available --matches '*OpenShift*'
+----
+
+. In the output for the previous command, find the pool ID for an {product-title} subscription and attach the subscription to the registered system:
++
+----
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Enable the repositories required by {pipelines-title}:
++
+----
+# subscription-manager repos --enable="pipelines-1_DOT_0-for-rhel-8-x86_64-rpms"
```

Comment:
I guess repo name should be something like this
`subscription-manager repos --enable="pipelines-1.0-for-rhel-8-x86_64-rpms"`

Reply:
done